src/grammar/ast.ts

```diff
@@ -320,6 +321,15 @@ export type AstStatementForEach = {
     loc: SrcInfo;
 };
 
+export type AstStatementDestruct = {
+    kind: "statement_destruct";
+    type: AstTypeId;
+    identifiers: Map<string, [AstId, AstId]>; // field name -> [field id, local id]
```

Comment:
```suggestion
    /** field name -> [field id, local id] */
    identifiers: Map<string, [AstId, AstId]>;
```

A tiny change to make that comment show up in hover info